packages/block-editor/src/components/block-list/block.js

```diff
@@ -422,6 +420,7 @@ function BlockListBlock( {
 	const shouldShowInsertionPoint =
 		( isPartOfMultiSelection && isFirstMultiSelected ) ||
 		! isPartOfMultiSelection;
+	const isDragging = isDraggingBlocks && ( isSelected || isPartOfMultiSelection );
```

Comment:
We don't show the block mover until a block is selected so this logic is fine. However, I'm not familiar with how we're going to update the toolbar and whether the drag icon is going to be shown on situations where the block isn't selected (on hover, etc). Flagging in case it makes you think this could be problematic.

Reply:
> he drag icon is going to be shown on situations where the block isn't selected

Right now, we won't be showing any UI when the block is hovered (other than bordders maybe). In fact we recently removed the movers from hover states. Thanks for flagging though :)